tests/components/imeon_inverter/test_config_flow.py

```diff
@@ -0,0 +1,148 @@
+"""Test the Imeon Inverter config flow."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.imeon_inverter.const import DOMAIN
+from homeassistant.const import CONF_ADDRESS
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from .conftest import TEST_ERROR, TEST_SERIAL, TEST_USER_INPUT
+
+from tests.common import MockConfigEntry
+
+
+@pytest.mark.usefixtures("mock_async_setup_entry")
```

Comment:
```suggestion
pytestmark = pytest.mark.usefixtures("mock_async_setup_entry")


```
sorry should've been clearer, otherwise it's not used for the entire module